### PR TITLE
ci: use auto-merge wrapper

### PR DESCRIPTION
**Pull request by jamacku**

### CODE REVIEW

**Comment by phracek** [comment]
[test-all]

**Review by phracek — COMMENTED** [review]
@jamacku The change looks good, only one think. The gather metadata is one test that succeded and therefore the CI is green, that is not true. Do you know, if it is possible to fix it?

**Comment by jamacku** [comment]
@phracek The problem is that you don't have any tests that would trigger on PR submission automatically. I currently don't see how to handle this from the auto-merge side.

But IMHO this is only a theoretical problem. In case when someone approves PR within seconds of its submission, then it is possible that it will be merged on the first run of auto-merge. But I think this will never happen.

Once the Schedule auto-merge is finished, you will get proper CI status in a comment.

**Review by phracek — APPROVED** [review]

File: .github/workflows/auto-merge-on-demand.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-name: Auto Merge On Demand
+name: Auto Merge Scheduled / On Demand
 on:
   schedule:
-    # Workflow runs every 45 minutes
-    - cron: '*/45 * * * *'
+    # Workflow runs every 30 minutes
```
[the remark this answers is not in the document]
Comment (jamacku) [reply .github/workflows/auto-merge-on-demand.yml]: np, updated

**Comment by phracek** [comment]
[test]

**Pull request merged**